src/authInfo.ts

```diff
@@ -255,11 +255,15 @@ function parseIdUrl(idUrl: string) {
 //
 // Ideally, this would be removed at some point in the distant future
 // when all auth files now have the clientId stored in it.
-const DEFAULT_CONNECTED_APP_INFO = {
+const LEGACY_DEFAULT_CONNECTED_APP_INFO = {
   legacyClientId: 'SalesforceDevelopmentExperience',
   legacyClientSecret: '1384510088588713504',
 };
 
+export const DEFAULT_CONNECTED_APP_INFO = {
```

Comment:
I feel like the LEGACY in the above is redundant. You could have just had clientIf in the above object, so `DEFAULT_CONNECTED_APP_INFO.clientId` and `DEFAULT_CONNECTED_APP_INFO.legacyClientId`.